wpeverest/user-registration
# Enhance - Conditional logic in edit profile

### All Submissions:






### Changes proposed in this Pull Request:
The conditional logic was not working in the edit profile endpoint of my account page. This PR will make conditional logic to be able to work in the edit profile section.

### How to test the changes in this Pull Request:

1. Navigate to the form builder and enable conditional logic and rule for a field.
2. Navigate to the edit profile section of my account.
3. Check if the conditional logic works for that field.

### Types of changes:




### Other information:




### Changelog entry

> Enhance - Conditional logic in edit profile
